frontend/server/src/Controllers/Problem.php

```diff
@@ -48,7 +48,8 @@ class Problem extends \OmegaUp\Controllers\Controller {
         'problemRestrictedTagKarel',
         'problemRestrictedTagLanguage',
         'problemRestrictedTagOnlyOutput',
-        'problemRestrictedTagInteractive'
+        'problemRestrictedTagInteractive',
+        'wordsNoSubmissions'
```

Comment:
```suggestion
        'problemRestrictedTagNoSubmissions'
```

no podemos restringir un tag que no empiece con `problemRestrictedTag`. así que habría que crear esa cadena de traducción.